desc/coils.py

```diff
@@ -1298,8 +1388,15 @@ class MixedCoilSet(CoilSet):
     ----------
     coils : Coil or array-like of Coils
         collection of coils
-    currents : float or array-like of float
-        currents in each coil, or a single current shared by all coils in the set
+    NFP : int (optional)
```

Comment:
removed NFP and sym from docstring


call super init with the default values here